client/models/session.js

```diff
@@ -0,0 +1,6 @@
+import Backbone from 'backbone'
+import { data as sd } from 'sharify'
+
+export default class Session extends Backbone.Model {
```

Comment:
seeing backbone not written in coffeescript is weirddd

Reply:
haha true, let's not make a habit of it though :)